bot.py

```diff
@@ -8,36 +8,52 @@
 load_dotenv()
 
 GITHUB_TOKEN = os.getenv('GITHUB_TOKEN')
-g = Github(GITHUB_TOKEN)
+git = Github(GITHUB_TOKEN)
 
 DISCORD_TOKEN = os.getenv('DISCORD_TOKEN')
 bot = commands.Bot(command_prefix='!git ')
 
-user = g.get_user()
+user = git.get_user()
 
 @bot.command()
 async def hello(ctx): #!git hello
 	await ctx.send('Hi there '+ctx.message.author.name)
 
-@bot.command()
+
+@bot.command(brief="brief summary of repo: number of issues, prs, stars, etc. Args: repoName")
+async def summary(ctx, repoName): 
+	repo = git.get_repo(repoName)
+	pulls = repo.get_pulls(state='open', sort='created').totalCount
+	issues = repo.get_issues(state='open').totalCount - pulls
+	contributors = repo.get_contributors().totalCount
+	stargazers = [ s for s in repo.get_stargazers() ]
+	await ctx.send(
+		"Stars: "+str(len(stargazers)) + '\n'
+		"Contributors: "+str(contributors) + '\n'
+		"Open issues: "+ str(issues)+'\n'+
+		"Open pull requests: "+str(pulls)+'\n'
```

Comment:
Bit of a nit: you can also make these a list of strings and [`join`](https://www.w3schools.com/python/ref_string_join.asp) them using a new line like `"\n".join(["a", "list", "of", "strings"])` instead of adding a `\n` several times

Reply:
yeah that makes sense.